### PR TITLE
Copy terraform files into results directory

### PR DESCRIPTION
**Pull request by bryan-bar**
Instead of saving the terraform files into the results directory, we will be copy any needed files into it and do a recursive upload of the results directory to the s3 bucket.

Execution bit set for the following scripts:
`aws-pgbench-postgresql/00_validate/run.sh`
`aws-tpcc-pgd/00_validate/run.sh`
`aws-tpcc-refdata/00_validate/run.sh`

Ref: https://github.com/EnterpriseDB/edb-benchmarks/pull/76

### CODE REVIEW

File: aws-dbt2-aurora/05_push_results/run.sh [hunk 1]
```diff
@@ -1,22 +1,7 @@
 #!/bin/bash -eux
 # Push DBT2 files to the S3 bucket
-
-TERRAFORM_PROJECT_PATH="${RESULTS_DIRECTORY}/${TERRAFORM_PROJECT_NAME}"
-cd $RESULTS_DIRECTORY/dbt2-data
+cd "$RESULTS_DIRECTORY"
```
Comment (mw2q) [inline aws-dbt2-aurora/05_push_results/run.sh:3]: I think we should remove this `cd` and put the `RESULTS_DIRECTORY` in the aws command.

**Review by mw2q — CHANGES_REQUESTED** [review]
Looks good.  I do think the `cd` step in the results handling can be simplified that extra little bit. :)

File: aws-dbt2-aurora/05_push_results/run.sh (hunk 1 above)
Comment (bryan-bar) [reply aws-dbt2-aurora/05_push_results/run.sh]: `cd` removed from step and handled through `s3` command instead

File: aws-hammerdb-epas-oracle-compatibility/04_push_results/run.sh [hunk 2]
```diff
@@ -1,15 +1,7 @@
 #!/bin/bash -eux
 # Push HammerDB files to the S3 bucket
-TERRAFORM_PROJECT_PATH="${RESULTS_DIRECTORY}/${TERRAFORM_PROJECT_NAME}"
-cd "${RESULTS_DIRECTORY}/tprocc-data"
+cd "${RESULTS_DIRECTORY}"
```
Comment (mw2q) [inline aws-hammerdb-epas-oracle-compatibility/04_push_results/run.sh:3]: found a lingering `cd` :)

**Review by mw2q — APPROVED** [review]

File: aws-hammerdb-epas-oracle-compatibility/04_push_results/run.sh (hunk 2 above)
Comment (bryan-bar) [reply aws-hammerdb-epas-oracle-compatibility/04_push_results/run.sh]: removed

**Pull request merged**